src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -928,6 +1158,12 @@ second echos are available.
 
 Required fields:
 
+{{ MACROS___make_metadata_table(
+   {
+      "EchoTime": "REQUIRED",
```

Comment:
This needs to just be a number, not a number or array of numbers.
![Screenshot from 2021-04-28 12-34-23](https://user-images.githubusercontent.com/83442/116440023-146b6580-a81e-11eb-9720-91dd4c0e24da.png)

Reply:
Maybe we can include a caveat in the description here? EchoTime in the general sense can be a number or array of numbers, but obviously that's not really applicable to field maps.

Reply:
Yeah, I'm not sure what the best move is here. It feels weird to say "Note that much of the preceding does not apply here." What if we made this `"fmapEchoTime": "REQUIRED"` with a modified type and description?

Reply:
Wouldn't that be introducing a new field? What about updating the base definition to say something like,

> This field MAY be an array when it references a 4D imaging file in which volume timing is critical for interpretation, such as in [ASL](/04-modality-specific-files/01-magnetic-resonance-imaging-data.html#arterial-spin-labeling-perfusion-data) or variable echo time fMRI sequences.

Reply:
I assumed `name` was what was going into the field, not the filename... This makes it a bit harder to customize.

The problem with changing that last bit of text is that it was pretty heavily workshopped during the ASL/qMRI discussions to satisfy everybody.

Reply:
>I assumed name was what was going into the field, not the filename... This makes it a bit harder to customize.

I honestly haven't leveraged `name` at all so far. I don't want to rely too heavily on `name` and the filename not matching, but it could be very useful in special cases, which this might count as.

>The problem with changing that last bit of text is that it was pretty heavily workshopped during the ASL/qMRI discussions to satisfy everybody.

Good point.

Reply:
I've updated the code to do as you requested, and it should be good as of 3e2a56d.